starlark/build.rs

```diff
@@ -50,18 +95,29 @@ fn conformance_test_cases(path: &str) {
         let path = path_entry.strip_prefix(&base).unwrap().to_str().unwrap();
         if path_entry.extension().unwrap().to_str().unwrap() != "md" {
             // Exclude markdown files
-            write!(
-                outfile,
-                r#"
+            let content = match test_or_bench {
```

Comment:
```rust
let content = format!(match test_or_bench {
  TestOrBench::Test => r#"
#[test]
fn test_{}() {{
    do_conformance_test("{}")
}}
"#,
  TestOrBench::Bench => r#"
#[bench]
fn bench_{}(bencher: &mut Bencher) {{
    do_bench(bencher, "{}")
}}
"#,
},
                    path_entry.file_stem().unwrap().to_str().unwrap(),
￼	                    path
￼	                );
```

Or something like that should reduce the number of lines. Maybe even pass the templating format directly as function argument?

Reply:
The first parameter of the `format` function must be compile time string, so it's not possible to do it as you suggested. I have extracted this snippet into `format_test_content` function.